source/manual/buckups.html.md

```diff
@@ -0,0 +1,112 @@
+---
+owner_slack: "#govuk-developers"
+title: Use of backups
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+type: learn
+---
+
+This manual provides a shared definition of a backup and documents our current use of backups summarising strategies and tools we use.
+
+> A backup is a process of creating a copy of the data stored in a database and storing it in a safe location to allow data recovery in case of a disaster, system failure or human error.
+
+# AWS RDS Backups
+
+## Point-in-time RDS backups
+
+Point-in-time backups for Amazon Relational Database Service (RDS) allow us to restore your databases to a specific point in time in the past. This feature is useful when we want to restore your database to a state before an unintended change or data loss occurred.
+
+When point-in-time backups for an RDS instance are enabled, in our case in the [terraform configuration](https://github.com/alphagov/govuk-aws/blob/main/terraform/modules/aws/rds_instance/main.tf), Amazon RDS automatically takes regular (daily) backups of the database and stores them in Amazon S3. 
+Each backup contains a snapshot of the database at a specific point in time. We can restore the database to any of these points in time, within the retention period which is set for 7 days.
```

Comment:
This looks right to me, though it took me a while to validate that. Perhaps we could add a bit more detail here:

1. `backup_retention_period` is set to 7 days, creating one snapshot per day during the specified `backup_window` (see  [aws_db_instance terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance)). The `backup_window`, by the way, should generally be set to a time where some increased latency would be acceptable (i.e. overnight), as [storage I/O may be suspended for a few seconds](https://shunliz.gitbooks.io/aws-certificate-notes/content/rds/rds-back-up-restore-and-snapshots.html) when the backup initialises, though this only seems to apply to single A/Z instances.
2. These snapshots are visible on the "[Automated backups](https://eu-west-1.console.aws.amazon.com/rds/home?region=eu-west-1#automatedbackups:)") page in the AWS console (and are the same snapshots as can be seen under each RDS instance's "Maintenance & backups" tab.). 
3. The "Earliest restorable time" and "Latest restorable time" columns show the time range that can be restored to: read [Restoring a DB instance to a specified time](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_PIT.html).

From that doc:

> RDS uploads transaction logs for DB instances to Amazon S3 every five minutes

I couldn't find the S3 bucket so I wonder if this is an 'internal' bucket only visible to AWS?

Reply:
I rewrote it to structure it around the attributes and included the points you mentioned. Also mentioned that all our RDS databases, except for Transition PostgreSQL, are Multi-AZ.

> I couldn't find the S3 bucket so I wonder if this is an 'internal' bucket only visible to AWS?

I couldn't find any info on this referring specifically o RDS backups but in the [DocumentDB docs](https://docs.aws.amazon.com/documentdb/latest/developerguide/backup_restore.html) it say
"These are service-managed Amazon S3 buckets and you will not have access to the backup files" so I'm wondering if this is also the case here.